controller/iteration_blackbox_test.go

```diff
@@ -424,15 +472,32 @@ func (rest *TestIterationREST) TestSuccessUpdateIteration() {
 	assert.Equal(rest.T(), 0, updated.Data.Relationships.Workitems.Meta[KeyTotalWorkItems])
 	assert.Equal(rest.T(), 0, updated.Data.Relationships.Workitems.Meta[KeyClosedWorkItems])
 
-	// try update using some other user
-	otherIdentity := &account.Identity{
-		Username:     "non-space-owner-identity",
-		ProviderType: account.KeycloakIDP,
+	// update iteration using Collaborator
```

Comment:
I would prefer that this test be splitted in 2 parts if the goal is to verify that the iteration can be edited by users with different roles (`owner` vs `collaborator`)

Reply:
yes! done in https://github.com/fabric8-services/fabric8-wit/pull/1783/commits/80ae86445281239f604d0a038eba40ff8b347607